packages/react-components/react-field/etc/react-field.api.md

```diff
@@ -146,436 +146,715 @@ export type TextareaFieldProps = FieldProps<typeof Textarea>;
 
 // @public
 export const useField_unstable: <T extends FieldComponent>(props: Omit<Partial<FieldSlots<T>>, never> & ("ref" extends keyof Exclude<T extends React_2.ComponentType<infer Props> ? (Props extends {
-    children?: unknown;
+    children?: unknown; /**
+    * Merge two possibly-undefined IDs for aria-describedby. If both IDs are defined, combines
+    * them into a string separated by a space. Otherwise, returns just the defined ID (if any).
+    */
 } ? Props : Props & {
```

Comment:
API extractor is blowing up on these types for some reason. This is fixed in a separate PR: #24703